src/views/SignDid/SignDid.tsx

```diff
@@ -52,11 +51,9 @@ export function SignDid({ identity, popupData, onCancel, credentials }: Props) {
       const { didDocument, authenticationKey } =
         await getIdentityCryptoFromSeed(seed);
 
-      const signature = Utils.Crypto.u8aToHex(
-        authenticationKey.sign(plaintext),
-      );
+      const signature = Crypto.u8aToHex(authenticationKey.sign(plaintext));
       const didKeyUri =
-        `${didDocument.uri}${didDocument.authentication[0].id}` as DidResourceUri;
+        `${didDocument.id}${didDocument.authentication?.[0]}` as DidUrl;
```

Comment:
The authentication key is optional now?

Reply:
Apparently

Reply:
Maybe @rflechtner can give us some insight

Reply:
I expect the reason is that the DID specification says authentication keys are optional

Reply:
Yeah resolving a deactivated DID still returns a DID document, but an empty one (only the id property is there).

Reply:
Feels like purity has beat practicality in this case